test/js/web/fetch/fetch.test.ts

```diff
@@ -1816,3 +1816,84 @@ it("304 not modified with 0 content-length does not cause a request timeout", as
   expect(await response.arrayBuffer()).toHaveLength(0);
   server.stop(true);
 });
+
+describe("http/1.1 response body length", () => {
+  // issue #6932 (support response without Content-Length and Transfer-Encoding) + some regression tests
+
+  let server: net.Server | undefined;
+  beforeAll(async () => {
+    server = net.createServer(socket => {
```

Comment:
Could we change this to use `Bun.listen()` instead of `net.createServer()`? The former will be faster, and it's consistent with the rest of the tests.

See line 1798 for an example.